vagrant/boxes.d/30-source.yaml

```diff
@@ -41,20 +41,91 @@ pulp2-nightly-pulp3-source-centos7:
     host_path: '..'
     guest_path: '/home/vagrant/devel'
     reverse: False
-  memory: 10240
+  # TODO before merge:
+  # memory: 10240
+  memory: 4608
   cpus: 4
   ansible:
     playbook:
       - "pulp-ci/ci/ansible/pulp_server.yaml"
       - "vagrant/playbooks/source-install.yml"
     variables:
-      # The pulp2 playbook is missing more fine graned privilege control
+      # The pulp2 playbook is missing more fine-grained privilege control
       ansible_become: true
       pulp_build: 'nightly'
       pulp_version: '2.21'
       pulp_webserver_server: 'apache'
+      pulp_devel_install_podman: false
     galaxy_role_file: "requirements.yml"
 
+# Update the Travis scripts whenever you update the RAM/CPU values
+pulp2-nightly-pulp3-source-fips-a:
+  box: 'centos8-fips'
+  sshfs:
+    host_path: '..'
+    guest_path: '/home/vagrant/devel'
+    reverse: False
+  # TODO before merge:
+  # memory: 10240
+  memory: 4608
+  cpus: 4
+  ansible:
+    playbook:
+    #  - "vagrant/playbooks/source-install.yml"
+      - "vagrant/playbooks/host-pulp2.yml"
+    variables:
+      # The pulp2 playbook is missing more fine-grained privilege control
+      pulp_webserver_server: 'apache'
+    galaxy_role_file: "vagrant/requirements.nested-docker.yml"
+
+# Update the Travis scripts whenever you update the RAM/CPU values
+pulp2-nightly-pulp3-source-fips-b:
+  provider: docker
+  docker_options:
+    #build_dir: vagrant
+    image: quay.io/mikedep333/vagrant-pulp2:latest
+    cmd:
+      - /usr/sbin/init
+    # cmd:
+    #  - sh
+    #  - -c
+    #  - "/usr/bin/yum install -y openssh-server systemd systemd-sysv && exec /usr/sbin/init"
```

Comment:
should we delete it?

Reply:
Deleted, this was prior to me determining that we must use a Dockerfile and pre-built image.